### PR TITLE
added poster package

### PR DESCRIPTION
**Pull request by HassanAmr**
Hi,

Kindly consider adding this package to conda-forge as it will help creating conda packages that depend on it.

Best regards,
Hassan

### CODE REVIEW

File: recipes/poster/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+about:
+  home: https://atlee.ca/software/poster/
+  license: MIT
+  license_family: MIT
+  summary: Streaming HTTP uploads and multipart/form-data encoding
+
+extra:
+  recipe-maintainers:
+    - hassanamr
```
Comment (bgruening) [inline recipes/poster/meta.yaml]: feel free to add me as maintainer

Comment (synapticarbors) [inline recipes/poster/meta.yaml]: @bgruening -- I just merged this PR, but @HassanAmr can certainly add you ad a co-maintainer via a PR to the feedstock.

**Comment by synapticarbors** [comment]
@HassanAmr -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

File: recipes/poster/meta.yaml (hunk 1 above)
Comment (HassanAmr) [inline recipes/poster/meta.yaml]: @synapticarbors, thanks :)
@bgruening, I will add you as soon as I get the notification for if being on feedstock. Thanks :)

**Pull request merged**